apps/sms/desktop-mock/navigator_moz_mobilemessage_shim.js

```diff
@@ -7,12 +7,12 @@
 
 *********************************************************** */
 (function(window) {
- 
-  if (!parent.navigator.mozMobileMessage) {
+
+  if (!opener || !opener.navigator.mozMobileMessage) {
```

Comment:
ok, in that case, I think we should not use this shim but the real mock; because the opener might just eventually disappear.

but maybe we should just use the mock everywhere -- after all we're in the desktop here, it's just for mocking.